core/src/main/java/io/kestra/core/services/AbstractService.java

```diff
@@ -0,0 +1,86 @@
+package io.kestra.core.services;
```

Comment:
```suggestion
package io.kestra.core.server;
```

This is not a service but an abstract class for servers (that implements a Service interface but we may rename this interface)